cmd/doc/main.go

```diff
@@ -54,6 +54,11 @@ var (
 	portEnv     = "PG_PORT"
 	dbEnv       = "PG_DB"
 
+	gitterHostEnv     = "GITTER_HOST"
+	gitterPortEnv     = "GITTER_PORT"
+	gitterHostDefault = "127.0.0.1"
+	gitterPortDefault = "1234"
```

Comment:
Code updated below to use environment variables (so that this code can run in a docker aside the gitter runner), and these defaults were kept for backwards compatibility.